MaxText/max_utils.py

```diff
@@ -61,6 +61,18 @@ def calculate_num_params_from_pytree(params):
   assert total_parameters >= 0
   return total_parameters
 
+
+def calculate_total_params_per_chip(params):
+  def calculate_params_per_chip(pytree):
```

Comment:
Why the argument to `calculate_params_per_chip` called `pytree`? Should be `calculate_params_per_chip_leaf` and the argument should be `arr` or `leaf` or something.

Reply:
Done, thanks.